app/src/main/res/values/strings.xml

```diff
@@ -499,6 +499,9 @@ Signature: %5$s</string>
 
     <string name="transport_action_usage">Usage</string>
     <string name="transport_help_text">Tap your favorite transportation lines to highlight them. Tap again to revert them back to normal.</string>
+    <string name="mvv_widget" translatable="false">MVV Widget TUM</string>
+    <string name="mvv_widget_select_station">Select station</string>
+    <string name="mvv_widget_no_station">No station selected</string>
```

Comment:
so the is german and english fine for now or should i use the translators magic?